cmd/minishift/cmd/start.go

```diff
@@ -239,8 +239,7 @@ func applyAddOns(ip string, routingSuffix string, ocPath string, kubeConfigPath
 	addOnManager := addon.GetAddOnManager()
 	err := addOnManager.Apply(addon.GetExecutionContext(ip, routingSuffix, ocPath, kubeConfigPath, sshCommander))
 	if err != nil {
-		fmt.Println("Error executing addon commands: ", err)
-		atexit.Exit(1)
+		atexit.ExitWithMessage(1, fmt.Sprint("Error executing addon commands: ", err))
```

Comment:
Could you either squash this commit or use the same issue prefix in the commit message indicating in which context we've done this. Either works for me.